features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Base/Header/headersearch/SearchUtils.jsx

```diff
@@ -95,8 +119,12 @@ function getSuggestionValue(suggestion) {
 function getSuggestions(value) {
     const inputValue = value.trim().toLowerCase();
     const inputLength = inputValue.length;
-    const promisedAPIs = API.all({ query: { name: inputValue }, limit: 8 });
-    return inputLength === 0 ? new Promise(resolve => resolve([])) : promisedAPIs;
+    if (inputLength === 0 || (inputValue.includes(':') &&
+        (inputValue.trim().split(':').filter(x => x)).length === 1)) {
+        return new Promise(resolve => resolve({ obj: { list: [] } }));
```

Comment:
According to this logic, This will bypass the network call iff there is only one empty attribute search 

for example:
Following is workign :+1: 
```
name:sample version:
```

following is not working :-1: 
```
name: version:
```

Reply:
Fixed with a regex